src/components/Graph/Graph.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import Node from './Node';
+import Edge from './Edge';
+
+const ListItem = ({ items, renderItem }) => {
+  return items.map((item, index) => renderItem(item, index));
+};
+
```

Comment:
This is the entire list, not listItem. 
If you want you can create a List component and use it just as you did (pass items and renderItem to the list)